Makefile

```diff
@@ -285,25 +285,14 @@ test-driver-adapter-planetscale: test-planetscale-vitess8
 build-qe-napi:
 	cargo build --package query-engine-node-api
 
-build-connector-kit-js: build-driver-adapters symlink-driver-adapters
-	cd query-engine/driver-adapters/connector-test-kit-executor && pnpm i && pnpm build
+build-connector-kit-js: build-driver-adapters
+	cd query-engine/driver-adapters && pnpm i && pnpm build
 
 build-driver-adapters: ensure-prisma-present
 	@echo "Building driver adapters..."
-	@cd ../prisma && pnpm --filter "*adapter*" i && pnpm --filter "*adapter*" build
+	@cd ../prisma && pnpm --filter "*adapter*" i
```

Comment:
@millsp I observed that if I don't install the deps from prisma, I get an error if running `pnpm build` from prisma-engines/query-engine/driver-adapters. Why can this be, given that all the projects that I'm trying to build are part of the workspace?

Reply:
I guess it is because they are still two different workspaces (since they have two different workspace files), so pnpm will not recursively install once it exits on the other workspace.